WalletWasabi/WabiSabi/Models/MultipartyTransaction/MultipartyTransactionParameters.cs

```diff
@@ -11,45 +11,4 @@ public record MultipartyTransactionParameters
 {
 	// version, locktime, two 3 byte varints are non-witness data, marker and flags are witness data.
 	public static int SharedOverhead = 4 * (4 + 4 + 3 + 3) + 1 + 1;
```

Comment:
It seems like this file can be removed entirely, by moving the static to somewhere else. Your call if you do it now or in next PR.

Reply:
Sure. Let's keep it for now. We can remove it as the last step.